src/Mailer/Ownership/SendMailsForOwnership.php

```diff
@@ -20,37 +20,38 @@
 use CultuurNet\UDB3\User\UserIdentityResolver;
 use DateTimeInterface;
 use Psr\Log\LoggerInterface;
+use Twig\Environment as TwigEnvironment;
 
 final class SendMailsForOwnership implements EventListener
 {
     private const SUBJECT_OWNERSHIP_REQUESTED = 'Beheers aanvraag voor organisatie {{ organisationName }}';
 
+    private ResqueCommandBus $mailerCommandBus;
```

Comment:
It's better to typehint on the `CommandBus` interface rather than the `ResqueCommandBus`. This couples the `SendMailsForOwnership` class too tightly to the Resque command bus, and it doesn't actually matter for the class which kind of bus it sends its commands to.

Reply:
I changed it to a ContextDecoratedCommandBus, because it needs a context to run as a worker in Resque.